lib/libzpool/uzfs_io.c

```diff
@@ -48,14 +49,9 @@
 
 #define	WRITE_METADATA(zv, metablk, mdata, tx)			\
 	do {							\
-		rl_t *mrl;					\
```

Comment:
this will be good if offset and len are multiple of zv_metavolblocksize. And, uzfs also recevies IOs in same way from tgt.
So, please add VERIFY in read and write APIs for offset & len to be multiple of zv_metavolblocksize

Reply:
ok